src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/CommonFunction.java

```diff
@@ -24,8 +29,25 @@ public static String nullToEmpty(Object o) {
         getFirstMapValue = (listMap) ->
         listMap.stream().findFirst().get();
 
-    public static boolean addressLineOneAndPostCodeAreBothNotEmpty(Map<String, String> address) {
-        return  ObjectUtils.isNotEmpty(address) && StringUtils.isNotBlank(address.get("AddressLine1"))
-                && StringUtils.isNotBlank(address.get("PostCode"));
+    public static boolean addressLineOneAndPostCodeAreBothNotEmpty(Map address) {
+        return  ObjectUtils.isNotEmpty(address)
+                && StringUtils.isNotBlank((String) address.get("AddressLine1"))
+                && StringUtils.isNotBlank((String) address.get("PostCode"));
```

Comment:
you could use uk.gov.hmcts.reform.bsp.common.mapper.AddressMapper.Field#POSTCODE const here